src/devices/Ws28xx/samples/LEDStripSample/Program.cs

```diff
@@ -0,0 +1,404 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Device.Spi;
+using System.Drawing;
+using Iot.Device.Ws28xx;
+using LEDStripSample;
+
+Console.WriteLine("Attach debugger if you want to, press any key to continue");
+Console.ReadKey();
+Console.Clear();
+
+SpiConnectionSettings settings = new(0, 0)
+{
+    ClockFrequency = 2_400_000,
+    Mode = SpiMode.Mode0,
+    DataBitLength = 8
+};
+
+using SpiDevice spi = SpiDevice.Create(settings);
+
+var ledCount = -1;
+MenuId menuId = MenuId.Root;
+Animations? effects = null;
+
+while (ledCount == -1)
+{
+    Console.Write("Number of LEDs (0 to exit): ");
+    var input = Console.ReadLine();
+    if (int.TryParse(input, out ledCount))
+    {
+        break;
+    }
+}
+
+if (ledCount == 0)
+{
+    return;
+}
+
+while (true)
+{
+    Console.Write($"{Environment.NewLine}1: WS2812b{Environment.NewLine}2: WS2815b{Environment.NewLine}3: SK68012{Environment.NewLine}0: Exit{Environment.NewLine}Type of Strip: ");
```

Comment:
I think this will be way more readable if you just do separate Console.WriteLine for each of these

Reply:
OK